README.md

```diff
@@ -1,6 +1,130 @@
 # GitHub webhook notifications 
 
-A simple webhook server forward message to telegram.
+A simple webhook server that helps you forward GitHub webhook messages to Telegram.
+
+
+
+## Compile
+
+It consumes around 1.2GiB of RAM at maximum, together with a disk usage of 2GiB. 
+
+**Please make sure you have abundant resources.**
+
+And you need an available rust compiler, `rustup`, for instance.
+
+```sh
+git clone https://github.com/KunoiSayami/github-webhook-notification.rs.git
+cd github-webhook-notification.rs
+cargo build --release
+```
+
+Then go to `./target` and you will find the executable binary file. Copy it to the place you want to destinate it to.
+
+
+
+## Install From Pre-built Executable
```

Comment:
This chapter is actually not very necessary

Reply:
But you know that some instances are not qualified for compiling the source code.
Sometimes they do need a prebuilt release.

Reply:
You can just place a link here, they should know what to do.

Reply:
like `You can get pre-build executable here: balabala`

Reply:
OK I got this.